lib/modules/manager/gomod/extract.ts

```diff
@@ -46,7 +50,7 @@ function getGoDep(lineNumber: number, goVer: string): PackageDependency {
     depType: 'golang',
     currentValue: goVer,
     datasource: GolangVersionDatasource.id,
-    versioning: 'go-mod-directive',
+    versioning,
```

Comment:
```suggestion
    ...(versioning && {versioning}),
```
only add if defined. we don't like undefined

Reply:
Nice!